test/splitgraph/commands/test_es_aggregation_pushdown.py

```diff
@@ -59,9 +59,75 @@ def test_elasticsearch_aggregation_functions_only(local_engine_empty):
     # Assert aggregation result
     assert result[0] == (999, 25714.837, 49989, 25714837, 20, Decimal("30.171"))
 
+    # Test COUNT(*)
+    query = "SELECT COUNT(*) FROM es.account"
+
+    # Ensure query is going to be aggregated on the foreign server
+    result = get_engine().run_sql("EXPLAIN " + query)
+    assert _extract_queries_from_explain(result)[0] == {
+        "query": {"bool": {"must": []}},
+        "track_total_hits": True,
+        "aggs": {},
+    }
+
+    # Ensure results are correct
+    result = get_engine().run_sql(query, return_shape=ResultShape.ONE_ONE)
+    assert result == 1000
+
+
+@pytest.mark.mounting
+def test_simple_aggregation_functions_filtering(local_engine_empty):
+    _mount_elasticsearch()
+    query = """
+    SELECT avg(age), max(balance)
+    FROM es.account
+    WHERE balance > 20000 AND age < 30
+    """
+
+    # Ensure query is going to be aggregated on the foreign server
+    result = get_engine().run_sql("EXPLAIN " + query)
+    assert _extract_queries_from_explain(result)[0] == {
+        "query": {
+            "bool": {
+                "must": [{"range": {"balance": {"gt": "20000"}}}, {"range": {"age": {"lt": 30}}}]
+            }
+        },
+        "aggs": {
+            "avg.age": {"avg": {"field": "age"}},
+            "max.balance": {"max": {"field": "balance"}},
+        },
+    }
+
+    # Ensure results are correct
+    result = get_engine().run_sql(query)
+    assert len(result) == 1
+
+    # Assert aggregation result
+    assert result[0] == (Decimal("24.439862542955325"), 49795.0)
```

Comment:
Use something like `assert math.isclose(result[0][0], 24.4399, rel_tol=1e-05)` here just in case this slightly shifts with different versions of ES to make maintenance easier